### ISSUE TITLE
.rtdc file attributes

### ISSUE THREAD

**Issue by phidahl** [ZELLMECHANIK-DRESDEN/dclab#124]
There are some attributes that I'd like to add:

- online_contour:bg empty (boolean) as discussed here: https://github.com/ZELLMECHANIK-DRESDEN/ShapeIn_Issues/issues/5
- fluorescence:channel x baseline offset (integer) as discussed here: https://github.com/ZELLMECHANIK-DRESDEN/ShapeOut2/issues/81

The second one will have implications for plotting the traces in ShapeOut properly. Although in most cases the offset is very small compared to the signal, so that the plots would be off less than a pixel. For older dataset without these attributes the offset is already corrected in the traces stored in the dataset.

Secondly there are some attributes that have other types in .rtdc file as in definitions.py:

- fluorescence:sample rate (float) should be integer
- imaging: roi position x
- imaging: roi position y (float) should be integer

Are there implications when the type expected by dclab does not match the type found in the attribute? dclab-verify-dataset does not complain.

**Comment by paulmueller** [comment]
Thanks for the clarification regarding the dtypes. I will take a closer look and see if there would be any issues with the current tests. I assume I put "float" there because it is more general.

`dclab-verify-dataset` currently does not check for dtypes.

**Comment by paulmueller** [comment]
Please not that the baseline offset is already implemented since dclab 0.33.0 as `baseline ? offset`:

```
        ["baseline 1 offset", fint, "Baseline offset channel 1"],
        ["baseline 2 offset", fint, "Baseline offset channel 2"],
        ["baseline 3 offset", fint, "Baseline offset channel 3"],
```

**Comment by paulmueller** [comment]
There are no issues with the data types, but I have to rewrite some of the dataset checks because of the new `online_contour:bg empty` key.
